web/themes/new_weather_theme/templates/block/block--weathergov-weather-story-image.html.twig

```diff
@@ -1,26 +1,27 @@
-<div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2" wx-outer-radar-container>
-  <div class="grid-row grid-col-12 tablet-lg:grid-col-12 bg-accent-cool-lightest padding-x-2 border-top border-base-light">
-    <h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">{{ "From the forecaster" | t }}</h3>
-  </div>
-  <div class="grid-container padding-0 bg-accent-cool-lightest grid-col-12 tablet-lg:grid-col-12 border-bottom border-base-light">
-    <div class="grid-row padding-x-2 margin-bottom-2">
-      <h4 class="wx-visual-h3 text-primary-dark margin-0">{{ content.title }}</h4>
-    </div>
-    <div class="grid-row padding-x-2 margin-bottom-2">
-      <p class="margin-0">
+<div class="bg-accent-cool-lightest border-top-1px border-bottom-1px border-base-light margin-top-4 wx-weather-story-block-section">
+  <div class="grid-container tablet:padding-x-4 padding-x-2">
+  <div class="wx-weather-story-wrapper">
+    {# Top section with text #}
+    <div class="margin-bottom-2">
+      <h3 class="wx-visual-h2 text-normal text-primary-darker margin-top-3 margin-bottom-2">{{ "From the forecaster" | t }}</h3>
+      <h4 class="wx-visual-h3 text-primary-dark margin-0 margin-bottom-2">{{ content.title }}</h4>
+      <p class="margin-0 margin-bottom-2">
         <strong class="text-base-dark">Created: {{ content.updated.formatted }}</strong> by the <a class="usa-link" href="/offices/{{ content.wfo_code | normalize_wfo }}">{{ content.wfo_name }} WFO</a>
       </p>
-    </div>
-    <div class="grid-row padding-x-2 margin-bottom-2">
       <p class="margin-0">{{ content.description }}</p>
     </div>
-    <div class="grid-row margin-bottom-2">
-      {% if content.image %}
-        <img src="{{ file_url(content.image.uri) }}" alt="{{ content.image.alt | replace({'"': "&quot;"}) }}">
+
+    {# Image area #}
+    {% if content.image %}
+    <div class="display-block width-ful">
+      <img src="{{ file_url(content.image.uri) }}" alt="{{ content.image.alt | replace({'"': "&quot;"}) }}">
```

Comment:
Going through with voice over, it sounds like the alt text is a repeat of the description. Looking at the `WeatherStoryImageBlock.php` it does seem like they are the same field (`description`), in which case I think we should keep the alt text here as `""` so that it's not duplicated for screen readers.